templates/todo/projects/java-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -25,19 +25,15 @@ param logAnalyticsName string = ''
 param resourceGroupName string = ''
 param webContainerAppName string = ''
 param apimServiceName string = ''
+param apiAppExists bool = false
+param webAppExists bool = false
 
 @description('Flag to use Azure API Management to mediate the calls between the Web frontend and the backend API')
 param useAPIM bool = false
 
 @description('Id of the user or app to assign application roles')
 param principalId string = ''
 
-@description('The image name for the api service')
-param apiImageName string = ''
-
-@description('The image name for the web service')
-param webImageName string = ''
```

Comment:
We are explicitly removing the ability to set the container image at provision time.  This makes `azd deploy` the only way to update the container image which is consistent with how other service targets are deployed.